deepspeed/runtime/zero/config.py

```diff
@@ -301,6 +302,10 @@ class DeepSpeedZeroConfig(DeepSpeedConfigModel):
     Boolean indicating whether to use quantized zero gradients
     for efficient all_2_all_reduce comm
     """
+    zeropp_loco_param: Optional[Dict[str, Any]] = None
+    """
+    loco-zero++  params
```

Comment:
maybe having a brief introduction of two params loco needed and what is prefered default values

Reply:
Thanks for the advice, we already add the basic description here in the update.